scripts/sort-manifests.js

```diff
@@ -20,5 +20,5 @@ for (const manifestName of manifests) {
     return a.moduleName > b.moduleName ? 1 : -1;
   });
 
-  await writeFile(manifestPath, JSON.stringify(manifest, null, 2), 'utf8');
+  await writeFile(manifestPath, JSON.stringify(manifest, null, 2) + '\n', 'utf8');
```

Comment:
not sure why this is showing up in the diff 🤔 
any idea?

Reply:
That's weird, idk